test/lib/rest-2-integration.js

```diff
@@ -104,7 +104,7 @@ describe('RESTv2 integration (mock server) tests', () => {
     ['alertDelete', 'alert_del.symbol.price', ['symbol', 'price']],
     ['accountTrades', 'trades.BTCUSD.0.10.50.1', ['BTCUSD', 0, 10, 50, 1]],
     ['wallets', 'wallets'],
-    ['walletsHistory', 'wallets_hist.end', ['end']],
+    ['walletsHistory', 'wallets_hist.end.currency', ['end', 'currency'],
```

Comment:
You dropped an extra `]` here at the end

Reply:
fixing, sorry i haven't seen this comment